scripts/reloader.bash

```diff
@@ -1,54 +1,84 @@
-#!/bin/bash
+# shellcheck shell=bash
+
 BASH_IT_LOG_PREFIX="core: reloader: "
-pushd "${BASH_IT}" >/dev/null || exit 1
+pushd "${BASH_IT}" > /dev/null || exit 1
+
+_bash-it-load-sources() {
```

Comment:
Making this a function ruins all explicit `declare`s, `typeset`s, &c. Bash makes any variables set this way as local to the function, which I suspect is why `reloader.bash` is a script file in the first place.